src/py/mains/consensus.py

```diff
@@ -13,6 +13,23 @@
 falcon.free_consensus_data.argtypes = [ POINTER(falcon_kit.ConsensusData) ]
 
 
+def get_longest_reads(seqs, max_n_read, max_cov_aln, sort=True):
+    if sort:
+        seqs = seqs[:1] + sorted(seqs[1:], key=lambda x: -len(x))
+
+    longest_n_reads = max_n_read                    
+    if max_cov_aln > 0:
+        longest_n_reads = 0
+        seed_len = len(seqs[0])
+        read_cov = 0
+        for seq in seqs[1:]:
+            longest_n_reads += 1
+            read_cov += len(seq)
+            if read_cov/seed_len > max_cov_aln: 
+                break
+        longest_n_reads = min(longest_n_reads, max_n_read)
```

Comment:
If you are trying to update a global, just declare it `global` in this function.